src/platform/configuration/common/configurationService.ts

```diff
@@ -589,16 +640,86 @@ export namespace ConfigKey {
 	}
 
 	/**
-	 * Internal and debugging settings that should be hidden from users.
-	 *
-	 * Features should only be in this list temporarily, moving on to experimental to be accessible to early adopters.
-	*/
-	export namespace Internal {
+	 * Advanced experimental settings that are available for all users to configure.
+	 * These should only be in this list temporarily. Either should be made stable or removed.
+	 */
+	export namespace AdvancedExperimental {
```

Comment:
I think having such precise and many namespaces makes it painful for developers to refer to settings. Can we just move previously internal settings from Internal to outer namespace along with other public settings?